README.md

```diff
@@ -51,43 +73,51 @@ The plugin will emit an error if it detects that it it missed a `loadstart` even
 
 ## Developing an Integration
 
-Once you call `player.ads()` to initialize the plugin, it provides six interaction points (four events and two methods) which you can use in your integration.
-
-Here are the events that communicate information to your integration from the ads plugin:
+First you call `player.ads()` to initialize the plugin. Afterwards, the flow of interaction
+between your ad integration and contrib-ads looks like this:
+
+* Player triggers `play` (EVENT) -- This media event is triggered when there is a request to play your player.
+videojs-contrib-ads responds by preventing content playback and showing a loading spinner.
+* Integration triggers `adsready` (EVENT) -- Your integration should trigger this event on the player to indicate that
+it is initialized. This can happen before or after the `play` event.
+* Contrib Ads triggers `readyforpreroll` (EVENT) -- This event is fired after both `play` and `adsready` have ocurred.
+This signals that the integration may begin an ad break by calling `startLinearAdMode`.
+* Integration calls `player.ads.startLinearAdMode()` (METHOD) -- This begins an ad break. During this time, your integration
+plays ads. videojs-contrib-ads does not handle actual ad playback.
+* Integration triggers `ads-ad-started` (EVENT) - Trigger this when each individual ad begins. This removes the loading spinner, which otherwise stays up during the ad break. It's possible for an ad break
+to end without an ad starting, in which case the spinner stays up the whole time.
+* Integration calls `player.ads.endLinearAdMode()` (METHOD) -- This ends an ad break. As a result, content will play.
+* Content plays.
+* To play a Midroll ad, start an end an ad break with `player.ads.startLinearAdMode()` and `player.ads.endLinearAdMode()` at any time during content playback.
```

Comment:
Typo ~~start an end an ad break~~ start and end an ad break

Reply:
👍